NotebookAnalyzers/entities.py

```diff
@@ -11,14 +11,20 @@ class Notebook:
 
     def __init__(self, notebook_path: str):
         # Importing the notebook as a Python dictionary
-        with open(notebook_path) as f:
+        with open(config.data_path + notebook_path) as f:
             self.nb_dict = json.load(f)
 
         # Importing the notebook as a Python script
-        transform_notebook(ipynb_file=notebook_path, export_list=["py"])
+        transform_notebook(ipynb_file=config.data_path + notebook_path, export_list=["py"])
         with open(notebook_path.replace(".ipynb", ".py")) as f:
             self.script = f.read()
 
+    def get_code(self):
+        return self.script
+
+    def get_dict(self):
+        return self.nb_dict
```

Comment:
I'd suggest you to read the first two answers to [this SO post](https://stackoverflow.com/questions/2627002/whats-the-pythonic-way-to-use-getters-and-setters). For simple cases like this one, the Pythonic way to use getters and setters is not to use them at all... I know that it's quite counterintuitive, especially for those used to other languages (as I myself was).

Attributes of Python objects are not private (even when their name starts with an underscore `_`): Python does not prevent their access by any means.

From outer scopes, you can directly access the attributes of an object; they should be turned into _properties_ - having getter and setter methods - only if the related get and set operations need to do some preprocessing/postprocessing stuff.